### ISSUE TITLE
[Bug] Zelda TOTK graphic flicking problem

### ISSUE THREAD

**Issue by JackySorFor1999** [Ryujinx/Ryujinx#6925]
### Description of the issue

https://github.com/Ryujinx/Ryujinx/assets/130536517/0bbe0b0f-bf99-4761-8f4c-27217baa643f

When I play TOTK with my M2 MacBook Pro, the graphic just keep flicking. It shows some weird color flashing sometimes. I disable all the mods. The Ryujinx settings are at default. I don't not why is this happening.

### Reproduction steps

Open Zelda TOTK and run through the map

### Log file

[Uploading Ryujinx_1.1.1330_2024-06-09_20-14-32.log…]()

### OS

macOS 14.5

### Ryujinx version

1.1.1330

### Game version

1.2.1

### CPU

Apple M2 Pro 10 core

### GPU

Apple M2 Pro 16 core

### RAM

16GB

### List of applied mods

All mods are disabled.

**Comment by Cyph3rous** [comment]
I'm having this issue as well... have you found any remedies as of yet?

**Comment by JackySorFor1999** [comment]
> I'm having this issue as well... have you found any remedies as of yet?

Nope, I've tried to install some visual fix mod, install 1.1.2 version of TOTK, install older version of Ryujinx. None of them can help with this issue...

**Comment by jfrankpax8** [comment]
I am having the exact same issues with my Macbook air (M1 chip). The game is still quite playable, but the constant flickering and artifacts are certainly distracting. I am not using any mods and playing 1.2.1 as well.

**Comment by BigDanceMouse** [comment]
1.2.1, mac studio M1 Max, same issues, no mods or with 60 fps - doesn't matter

**Comment by RVZO6** [comment]
exact same specs as OP, experiencing similar issue. Seems to be an apple silicon specifc issue.

**Comment by fuser-invent** [comment]
Same on MacBook Pro M2 Max 32GB RAM. Runs pretty smooth in 4K 60fps despite that though, with only occasional frame rate drops when entering a new area.
